test/index.js

```diff
@@ -88,6 +88,10 @@ describe('Generic', function () {
       assert(eql(undefined, null) === false, 'eql(undefined, null) === false');
     });
 
+    it('doesn\'t crash on weakmap key error (#33)', function () {
+      assert(eql({}, null) === false, 'eql({}, null) === false');
+    });
+
```

Comment:
I think this test belongs in the `null` section instead of the `undefined` section.